spec/dummy/application.rb

```diff
@@ -12,6 +12,11 @@ class Application < Rails::Application
     if Rails.version.match?(/(6.1|7.0)/)
       config.active_record.legacy_connection_handling = false
     end
+
+    if Rails.gem_version >= Gem::Version.new("7.1")
```

Comment:
Ideally I would like to add `config.load_defaults Rails::VERSION::STRING.to_f` to this file and remove these conditional settings that only exist to address the deprecation warnings. However, when loading the defaults, the specs in the `spec/requests` directory fail.